features/step_definitions/page_content_steps.rb

```diff
@@ -0,0 +1,53 @@
+Then(/^I should see "(.+)"$/) do |text|
```

Comment:
Let me think about this one a little. These are mostly looking for text (supplied as parameters or such), but there might be a better way to organize them.